### PR TITLE
Creating test event for moving cases to DN Pronounced.

### PR DESCRIPTION
**Pull request by jvictorcf**
Creating test event for moving cases to DN Pronounced.

### CODE REVIEW

File: definitions/divorce/json/AuthorisationCaseEvent.json [hunk 1]
```diff
@@ -1693,6 +1693,13 @@
     "UserRole": "citizen",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/03/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "testDNPronounced",
+    "UserRole": "caseworker-divorce-courtadmin_betacitizen",
```
Comment (hosainnet) [inline definitions/divorce/json/AuthorisationCaseEvent.json]: I don't think this is a valid role

Comment (qzhou-hmcts) [inline definitions/divorce/json/AuthorisationCaseEvent.json]: caseworker-divorce-courtadmin_betacitizen?? what is this role for?

Comment (jvictorcf) [reply definitions/divorce/json/AuthorisationCaseEvent.json]: I was testing you. You passed.

Comment (jvictorcf) [reply definitions/divorce/json/AuthorisationCaseEvent.json]: I was testing you. You passed, too. :)

File: definitions/divorce/json/AuthorisationCaseEvent.json [hunk 2]
```diff
@@ -1693,6 +1693,13 @@
     "UserRole": "citizen",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/03/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "testDNPronounced",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
```
Comment (qzhou-hmcts) [inline definitions/divorce/json/AuthorisationCaseEvent.json]: citizen

Comment (jvictorcf) [reply definitions/divorce/json/AuthorisationCaseEvent.json]: cases are put into this state by caseworkers, in real life...

Comment (qzhou-hmcts) [reply definitions/divorce/json/AuthorisationCaseEvent.json]: I don't think you want the caseworker see this event on real life.
All the test events should be triggered by citizen as we have all the control on citizen access

Comment (jvictorcf) [reply definitions/divorce/json/AuthorisationCaseEvent.json]: oh ok. fair enough. didn't know that.
I'll change it.

Comment (hosainnet) [reply definitions/divorce/json/AuthorisationCaseEvent.json]: The problem is that as it stands, the caseworker will see it in the dropdown and be able to trigger it

Comment (jvictorcf) [reply definitions/divorce/json/AuthorisationCaseEvent.json]: fair enough. changed it.

**Pull request merged**